nntrainer/src/databuffer.cpp

```diff
@@ -26,11 +26,14 @@
 #include <condition_variable>
 #include <cstring>
 #include <databuffer.h>
+#include <databuffer_file.h>
+#include <databuffer_func.h>
```

Comment:
Q. is there a reason for this? I am concerning that having derived class in base class.

Reply:
That was erronous. Removed.